src/libraries/System.CodeDom/tests/System/CodeDom/Compiler/CSharpCodeGeneratorTests.cs

```diff
@@ -2599,7 +2599,7 @@ string Normalize(string s)
 
                 var s = new StringBuilder();
                 s.AppendLine();
-                s.AppendLine($"Expected: {Environment.NewLine}$@\"{Normalize(expected)}\"");
+                s.AppendLine($"Expected: {Environment.NewLine}$@\"{Normalize(LineEndingsHelper.Normalize(expected))}\"");
                 s.AppendLine($"Actual:   {Environment.NewLine}$@\"{Normalize(actual)}\"");
```

Comment:
does "actual" need normalizing too?

also, you could store LineEndingsHelper.Normalize(expected) in a local at the start of the method, to avoid potentially doing it twice.

Reply:
@danmoseley As far as I understood, `actual` is code generagred, so the newlines are all `Environment.NewLine`s already.